tensorflow/lite/micro/kernels/arc_mli/conv.cc

```diff
@@ -509,8 +516,6 @@ TfLiteStatus EvalMliQuantizedPerChannel(
           ops::micro::PrepareLocalTensor(out_slice.Sub(), &out_local);
           ops::micro::PrepareLocalTensor(in_slice.Sub(), &in_local);
         }
-
-        TF_LITE_ENSURE(context, !in_slice.Done());
```

Comment:
I suppose it was removed due to some changes in TFLM. But suppose such assert was put not "just because".  Does it buggy? any alternatives?

Reply:
You right, that was buggy.  In some unusual cases, for example when after slicing, last part is smaller then convolution kernel, it processes all the input and output and then warns that input is not "done". 
Example: case when input [6x6], kernel is [3x3] and stride is 2.
We discussed with Jacco, he proposed just to remove this assert, and I agree. Reason is the loop is done to the end of output tensor, which for me is enough and also for the input check it is hard to fix, so I decided it is reasonable to remove.